pages/_app.js

```diff
@@ -22,7 +22,10 @@ export default function App({ Component, pageProps }) {
   function handleNewWisdomSubmit(wisdom) {
     setCurrentLibrary([...currentLibrary, wisdom]);
   }
-
+  function handleEditWisdomSubmit(wisdom) {
+    const arrayToKeep = currentLibrary.filter((elem) => elem.id != wisdom.id);
```

Comment:
== and != are non-strict equality operators. You should avoid them 99% of the time.
Non-strict equality tries to use type coercion to convert both values to the same type: "3" == 3 is true, which is seldomly what you want.

Reply:
Oops, typo, thx